iot-hub/Samples/device/PnpDeviceSamples/readme.md

```diff
@@ -1,31 +1,58 @@
-# Plug And Play Device Samples
-
-These samples demonstrate how a [plug and play convention][pnp-convention] enabled device interacts with IoT hub, to:
+---
+page_type: sample
+description: "A set of samples that show how a device that uses the IoT Plug and Play conventions interacts with either IoT Hub or IoT Central."
```

Comment:
The sample does not interact with IoT Central

Reply:
I believe what it's trying to say is that the device sample can be used to send to IoT Hub or IoT Central. Even though this sample is specifically for the hub, it can be used for central. Correct?

Reply:
This sample is actually already geared to run with central, it would use the "dps" flow, as opposed to "connection string" flow: https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/Parameter.cs#L20

Reply:
We use these samples in the IoT Central docs to show how a PnP  enabled device can interact with an IoT Central application using the DPS flow: https://docs.microsoft.com/en-us/azure/iot-central/core/tutorial-connect-device?pivots=programming-language-csharp